receiver/oracledbreceiver/scraper.go

```diff
@@ -78,6 +80,99 @@ const (
 		      AND S.PLSQL_ENTRY_SUBPROGRAM_ID = P.SUBPROGRAM_ID 
 			  LEFT JOIN V$SQL Q ON S.SQL_ID = Q.SQL_ID 
 			  WHERE S.SQL_ID IS NOT NULL AND S.STATUS = 'ACTIVE'`
+	dbTimeReferenceFormat  = "2006-01-02 15:04:05"
+	sqlIDAttr              = "SQL_ID"
+	childAddressAttr       = "CHILD_ADDRESS"
+	childNumberAttr        = "CHILD_NUMBER"
+	sqlTextAttr            = "SQL_FULLTEXT"
+	oracleQueryMetricsData = `SELECT /* collector-query */
```

Comment:
what about put it into a embeded text file?

also, can we put the comment as a prefix? i am asking because it looks like check prefix would be easier if we later need to filter out some collector queries